docs/modules/user-guide/partials/proc_updating-the-devfile-library-schema.adoc

```diff
@@ -0,0 +1,13 @@
+[id="proc_updating-the-devfile-library-schema_{context}"]
+= Updating the devfile library schema
+
+Update the schema version to ensure that you develop with the latest code from devfile properties:
+
+. To fetch the latest `github.com/devfile/api` Go module and to update the schema saved under `pkg/devfile/parser/data`, run, `./scripts/updateApi.sh`.
```

Comment:
```suggestion
. To fetch the latest `github.com/devfile/api` Go module and to update the schema saved under `pkg/devfile/parser/data`, run `./scripts/updateApi.sh`.
```

Reply:
```suggestion
. Fetch the latest `github.com/devfile/api` Go module and update the schema saved under `pkg/devfile/parser/data`:
+
[source,terminal]
----
$ ./scripts/updateApi.sh
----
```
This formatting makes it easier to copy the command.

Reply:
Do they need to make the script executable? If so we might need to add a step like this one:
```
. Add execute permissions for everyone on this script:
+
[source,terminal]
----
$ chmod a+x r./scripts/updateApi.sh
----
````

Reply:
@yangcao77 I applied Rolfe's suggestion. What do you think of the extra step?

Reply:
I don't think the "update the schema" should be included in the doc. as I said, it's a doc for consumer, they should not need to update our repository.